spec/features/users/show_spec.rb

```diff
@@ -1,39 +1,45 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
+require "rails_helper"
 
-describe 'Users#show' do
+describe "Users" do
   let(:user_profile) do
     create(:user)
   end
 
-  before(:each) do
+  before do
     sign_in user if user
     visit "/users/#{user_profile.id}".dup
   end
 
-  context 'when no user logged in' do
+  context "when no user signed in" do
     let(:user) { nil }
 
-    it 'demands signup' do
-      expect(page).to have_content 'You need to sign in or sign up before continuing.'
-      expect(page).to_not have_content user_profile.name
+    it "demands signup" do
+      expect(page).to have_content "You need to sign in or sign up before continuing."
```

Comment:
Can we disable multiple expects for feature specs? I could find and fix this rule later, but these are expensive tests!

Reply:
It appears so.  Dropping this in fixed that issue:

```
RSpec/MultipleExpectations:
  Exclude:
    - 'spec/features/**/*'
```